### PR TITLE
Remove unmaintained precss plugin from Readme

### PR DESCRIPTION
**Pull request by paulshryock**
A fresh install of postcss and precss throws this error:

> Unknown error from PostCSS plugin. Your current PostCSS version is 8.3.6, but precss uses 7.0.36.

Precss seems to be no longer maintained, and I suggest removing it from the Readme. [Last commit](https://github.com/csstools/precss/commits/master) was 3 years ago.

### CODE REVIEW

**Comment by ai** [comment]
Let's replace it with `postcss-nested`

File: README.md [hunk 1]
```diff
@@ -243,8 +242,7 @@ Then create `postcss.config.js`:
 ```js
 module.exports = {
   plugins: [
-    require('autoprefixer'),
-    require('postcss-nested')
+    require('autoprefixer')
```
Comment (ai) [inline README.md:245]: Let's keep `postcss-nested` here to show how to use multiple plugins

Comment (paulshryock) [reply README.md]: Sorry that was a typo. Fixed.

**Comment by ai** [comment]
Thanks. It was a very useful change.

**Pull request merged**